### PR TITLE
zh-cn: Format /web/html using Prettier

### PR DESCRIPTION
**Pull request by queengooborg**
This PR formats the `/web/html` folder of the Simplified Chinese locale using Prettier.

### CODE REVIEW

File: files/zh-cn/web/html/element/wbr/index.md [hunk 1]
```diff
@@ -50,10 +50,12 @@ slug: Web/HTML/Element/wbr
 
 ## 示例
 
-*[Yahoo 代码规范](https://web.archive.org/web/20121105171040/http://styleguide.yahoo.com/) *推荐 [在标点之前为 URL 换行](https://web.archive.org/web/20121105171040/http://styleguide.yahoo.com/editing/treat-abbreviations-capitalization-and-titles-consistently/website-names-and-addresses)，以便避免将标点符号留在行尾，这会让读者将 URL 的末尾搞错。
+_[Yahoo 代码规范](https://web.archive.org/web/20121105171040/http://styleguide.yahoo.com/)_ 推荐 [在标点之前为 URL 换行](https://web.archive.org/web/20121105171040/http://styleguide.yahoo.com/editing/treat-abbreviations-capitalization-and-titles-consistently/website-names-and-addresses)，以便避免将标点符号留在行尾，这会让读者将 URL 的末尾搞错。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/wbr/index.md:53]: Let's also remove the spaces

```suggestion
[_Yahoo 代码规范_](https://web.archive.org/web/20121105171040/http://styleguide.yahoo.com/)推荐[在标点之前为 URL 换行](https://web.archive.org/web/20121105171040/http://styleguide.yahoo.com/editing/treat-abbreviations-capitalization-and-titles-consistently/website-names-and-addresses)，以便避免将标点符号留在行尾，这会让读者将 URL 的末尾搞错。
```

File: files/zh-cn/web/html/element/address/index.md [hunk 2]
```diff
@@ -57,17 +57,17 @@ slug: Web/HTML/Element/address
...
-    Mozilla Foundation<br>
-    331 E Evelyn Ave<br>
-    Mountain View, CA 94041<br>
-    USA
-  </address>
+<address>
+  You can contact author at
+  <a href="http://www.somedomain.com/contact"> www.somedomain.com</a>.<br />
+  If you see any bugs, please
+  <a href="mailto:webmaster@somedomain.com"> contact webmaster</a>.<br />
```
Comment (yin1999) [inline files/zh-cn/web/html/element/address/index.md:64]: Should we also remove the spaces here?

```suggestion
  <a href="http://www.somedomain.com/contact">www.somedomain.com</a>.<br />
  If you see any bugs, please
  <a href="mailto:webmaster@somedomain.com">contact webmaster</a>.<br />
```

Comment (queengooborg) [reply files/zh-cn/web/html/element/address/index.md:64]: Probably a good idea -- by the way, you're always welcome to modify my PRs directly if you wish to!

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**